renderlib/ImageXYZC.h

```diff
@@ -91,10 +91,10 @@ class ImageXYZC
   // +1 means do not flip, -1 means flip
   void setVolumeAxesFlipped(int x, int y, int z);
 
-  uint32_t sizeX() const;
-  uint32_t sizeY() const;
-  uint32_t sizeZ() const;
-  uint32_t maxPixelDimension() const;
+  size_t sizeX() const;
+  size_t sizeY() const;
+  size_t sizeZ() const;

```

Comment:
this was previously a source of overflow when these values can be all multiplied together. using a 64-bit type is good insurance against this